uv/mlflow/reporter.py

```diff
@@ -26,16 +29,34 @@ class MLFlowReporter(b.AbstractReporter):
   """Reporter implementation that logs metrics to mlflow."""
 
   def __init__(self):
-    pass
+    self._client = mlf.tracking.MlflowClient()
```

Comment:
Using a client here is ~5x faster on my machine. From the mlflow code, it appears that this avoids recreating the sqlalchemy engine for each logging call.

Reply:
amazing!